### ISSUE TITLE
Allow for further narrowing of the receiving connections with notify

### ISSUE THREAD

**Issue by luisa-beerboom** [OpenSlides/openslides-icc-service#100]
As far as I understand currently the `to_*` fields are handled as a checklist where only one has to be true for the person to receive the message.

There has however been cases where it would've been nice to be able to specify f.E. that a group of users (as in `to_users`) should only receive the message when they are currently in a specific meeting or channel (as in `to_meeting` or `to_channels`).

Would it be possible/practical to add something like that?

Perhaps there could be a new boolean parameter `conjunct` that, if `true` causes the message to only be sent to connections that fulfil all `to_*` conditions.

From the client POV there is a workaround, as one can just send the other conditions as part of the message and then see if they should apply when receiving. However it is still not nice that messages are sent to connections that do not need them.

What do you think @ostcar ?

**Comment by ostcar** [comment]
If it is necessary, then we have to add a feature like this. But I hope, we find another way. I would like to have the syntax add simple as possible. To add the attribute for this use case seems for much more complexity that does not bring so much new flexibility.

Could you tell me the use cases the client currently has that needs this change?

**Comment by luisa-beerboom** [comment]
First of all I don't really mind how this is implemented exactly, the suggestion was just an example

Secondly: There _currently_ is only one use case: The client uses the notify functionality to invite others to join the jitsi conference call.
The user is only supposed to receive the invitation if he is:
1. the target user
2. currently logged in and looking at the same meeting from which the invitation was sent

In this case the message is only really useful to the receiver when it is sent to just this user via just the channel of the selected meeting.
The target user has no need for the message if he isn't in the target meeting (and therefore not listening to the channel).
Any other user that listens to the channel has no use for the message since it isn't meant for him.

The best way around this currently is, as I wrote above, to address the message just to the user and make the receiver decide whether the message is relevant to him (Which is what I did in OpenSlides/openslides-client#2201 but it is not very beautiful)

As there is a workaround this is probaby more of a nice-to-have anyway

**Comment by ostcar** [comment]
I don't think this is relevant anymore. If it is, lets talk about this in the next meeting
